### PR TITLE
Fixed issue 135 in branch issue-135. Added folder bugs/135 to reprodu…

### PR DESCRIPTION
**Pull request by johnzoet**
…ce the bug using command: serverless client deploy

### Background

[reference to original issue or other discussion]

### Proposed changes

[list of features added and/or changed]

### Proposed reviewers (optional)

[@ mentions of other contributors]

### CODE REVIEW

**Comment by iskrzycki** [comment]
Thanks for saving life! I've just verified this PR and I can confirm, that it solves problem and should be merged as asap as possible imo :)

**Review by iskrzycki — APPROVED** [review]

**Review by iskrzycki — APPROVED** [review]
Works as a charm, but what about removing comment?

File: index.js [hunk 1]
```diff
@@ -3,7 +3,8 @@
 const path = require('path');
 const fs = require('fs');
 
-const Confirm = require('enquirer');
+// Added braces to Confirm to get it working with command: serverless client deploy
+const { Confirm } = require('enquirer');
```
[the remark this answers is not in the document]
Comment (Shereef) [reply index.js:7]: #144 was merged so you can test this in master and let us know please

**Comment by fernando-mc** [comment]
@johnzoet thanks again for the PR! I think a fix for this should be included in v3.0.0 but let me know if you have any questions and feel free to reopen if it's not!

@iskrzycki let me know when you get a chance to try v3.0.0 and let me know if that solves the issues.

**Pull request closed without merging**